app-modules/service-management/src/Filament/Resources/ServiceRequestResource/Pages/EditServiceRequest.php

```diff
@@ -99,12 +100,20 @@ public function form(Form $form): Form
                                         fn (ServiceRequest $record) => ServiceRequestType::withTrashed()
                                             ->whereKey($record->priority?->type_id)
                                             ->orWhereNull('deleted_at')
+                                            ->when(! auth()->user()->hasRole('authorization.super_admin'), function (Builder $query) {
+                                                $query->whereHas('managers', function (Builder $query): void {
+                                                    $query->where('teams.id', auth()->user()->teams()->first()?->getKey());
+                                                })->orWhereHas('auditors', function (Builder $query): void {
+                                                    $query->where('teams.id', auth()->user()->teams()->first()?->getKey());
+                                                });
```

Comment:
Let's remove the auditor check here. You should only be able to create ones you are a manager of.